tools/python/runfiles/runfiles.py

```diff
@@ -161,6 +165,51 @@ def EnvVars(self):
     """
     return self._strategy.EnvVars()
 
+  def CurrentRepository(self, frame = 1):
+    """Returns the canonical name of the caller's Bazel repository.
```

Comment:
Let's update this doc to point to some bzlmod docs about canonical vs nickname repository names.